testFiles/Apple/multilayer_examples/animals_00_multilayer_lsel.avif.txt

```diff
@@ -0,0 +1,36 @@
+Input image is animals_00 from the Apple test set, rotated to upright (i.e. 180 degrees) and downscaled to 2048x1536 so encoding is faster.
+
+AV1 is encoded using libaom and the following settings:
+
+rc_end_usage = AOM_Q
+g_error_resilient = 0
+g_lag_in_frames = 0
+save_as_annexb = 0
+aom_codec_control(codec, AOME_SET_NUMBER_SPATIAL_LAYERS, 2)
+aom_codec_control(codec, AV1E_SET_COLOR_RANGE, 1)
+
+Layer 0:
+flags = AOM_EFLAG_FORCE_KF
+aom_codec_control(codec, AOME_SET_CQ_LEVEL, 62)
+aom_codec_control(codec, AOME_SET_SCALEMODE, &{ AOME_ONETWO, AOME_ONETWO })
+
+Layer 1:
+flags = AOM_EFLAG_NO_REF_LAST2 | AOM_EFLAG_NO_REF_LAST3 |
+	AOM_EFLAG_NO_REF_GF | AOM_EFLAG_NO_REF_ARF |
+	AOM_EFLAG_NO_REF_BWD | AOM_EFLAG_NO_REF_ARF2 |
+	AOM_EFLAG_NO_UPD_GF | AOM_EFLAG_NO_UPD_ARF |
+	AOM_EFLAG_NO_UPD_ENTROPY;
+aom_codec_control(codec, AOME_SET_CQ_LEVEL, 3)
+aom_codec_control(codec, AOME_SET_SCALEMODE, &{ AOME_NORMAL, AOME_NORMAL })
+
+
+Stream is embedded in a HEIF container as two items.
+Item 1:
+	ispe = 1024x768
```

Comment:
Is this one meant to have the same dimensions in item 1 and item 0? Are they layers improving the quality?

Reply:
> Is this one meant to have the same dimensions in item 1 and item 0? Are they layers improving the quality?

No, they're meant to have different dimensions. Layer 0 is a downscaled base layer with very low quality, while layer 1 is a full-res enhancement layer with full quality.

The idea here was to test that a parser correctly implements `lsel`. If it's ignored (even though it's specified as essential), a parser will run into problems when decoding item 0 since the iloc contains both layers. So `ispe` will say 1024x768, but the decoded data will be layer 1 with dimensions 2048x1536.

Reply:
Could you fix the readme then?

Reply:
Oops, my bad. Must have read your comment too late at night. You're indeed right that the readme is incorrect. Will fix.